sandbox-web/src/main/java/com/exadel/sandbox/team5/error/RestExceptionHandler.java

```diff
@@ -44,6 +45,13 @@
         return new ResponseEntity<>(apiError, BAD_REQUEST);
     }
 
+    @ExceptionHandler(BadCredentialsException.class)
+    public ResponseEntity<Object> handleAuthExceptions(Exception ex, WebRequest request) {
+        log.error("Bad credentials Exception", ex);
```

Comment:
log record for base credentials exception is not required

Reply:
Ok remove it